keps/sig-node/1592-pod-level-resource-limits/kep.yaml

```diff
@@ -0,0 +1,47 @@
+---
+title: Pod-level resource limits
+kep-number: 1592
+authors:
+  - "@liorokman"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2020-03-03
+reviewers:
+  - "@thockin"
+approvers:
+#  - TBD
+prr-approvers:
+#  - TBD
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+#latest-milestone: "v1.19"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.21"
```

Comment:
Needs to be set to 1.24.

Should be updated

Reply:
Changed release version to `1.27`